test/CadlRanchProjects/authentication/api-key/src/Generated/ApiKeyClient.cs

```diff
@@ -41,7 +41,7 @@ protected ApiKeyClient()
         }
 
         /// <summary> Initializes a new instance of ApiKeyClient. </summary>
-        /// <param name="endpoint"> The <see cref="Uri"/> to use. </param>
+        /// <param name="endpoint"> Service host. </param>
```

Comment:
See the explanaiton in https://github.com/microsoft/typespec/pull/4693/files#r1798861924

Since it's just a doc change. Unless this is really an issue. I'm going to adopt the change.